### DISCUSSION TITLE
Potential integration with a suite of Rarible Protocol, RaribleX and a substrate-exclusive NFT marketplace

### DISCUSSION THREAD

**Discussion by ShankarWarang** [ringdao-community-guild/discussion#114] [Integration]
We (Edgeware DAO contributors) have been working with the Rarible team for a while to propose a suite of Rarible Protocol integration, RaribleX integration and a white-labelled NFT marketplace for all the EVM environments running within the Polkadot/Substrate ecosystem. 
Would Darwinia and Crab be interested in having the integration? (This wouldn't cost anything to Darwinia and Crab as the proposal will be proposed to the Polkadot treasury.)

**Comment by hackfisher** [comment]
Sure, we are interested in it! cc @nblogist @wuminzhe could provide support for integration.

**Reply by ShankarWarang** [reply]
Noted! We are currently confirming the intent with all the substrate chains having EVMs, followed by a brief on Polkadot forum.
We will soon set up a group for representatives from all the interested chains.
Hello @nblogist @wuminzhe 👋 (I know @nblogist already, small world!)

**Reply by nblogist** [reply]
Hey Shankar! Lovely seeing you here! Looking forward to working together again soon!
